host_vars/meet.adfc-intern.de.yml

```diff
@@ -0,0 +1,51 @@
+proxmox_id: "100"
+proxmox_template: debian-10.0-standard_10.0-1_amd64.tar.gz
+proxmox_host: "proxmox01.adfc-intern.de"
+proxmox_net:
+  net0:
+    name: eth0
+    gw: 88.198.32.30
+    ip: 88.198.32.212/32
+    ip6: 2a01:4f8:a0:32b9::4/64
+    gw6: 2a01:4f8:a0:32b9::3
+    bridge: vmbr0
+proxmox_ramsize_in_mb: 10240
+proxmox_cores: 9
+ssh_users_admins:
+  - svena
+  - georgs
+jitsi_meet_server_name: "{{ inventory_hostname }}"
+certbot_create_if_missing: true
+certbot_admin_email: "webmaster@{{ jitsi_meet_server_name }}"
+certbot_certs:
+  - domains:
+     - "{{ jitsi_meet_server_name }}"
+certbot_create_standalone_stop_services: []
+jitsi_meet_ssl_cert_path: "/etc/letsencrypt/live/{{ jitsi_meet_server_name }}/fullchain.pem"
+jitsi_meet_ssl_key_path: "/etc/letsencrypt/live/{{ jitsi_meet_server_name }}/privkey.pem"
+jitsi_meet_ipv6_enable: true
+jitsi_meet_customize_the_ui: true
+
+jitsi_meet_lang: 'de'
+jitsi_meet_appname: 'ADFC Jitsi-Meet-Konferenz-System'
+jitsi_meet_org_link: 'https://adfc.de'
+jitsi_meet_welcomepage_title: 'Sichere, Videokonferenzen des Allgemeinen Deutschen Fahrrad-Clubs'
+jitsi_meet_welcomepage_description: 'Herzlich Wilkommen im Testbetrieb des Konferenzservers des ADFC. Weitere Infos: info@hamburg.adfc.de'
+
+# Auf den Unterseiten. Nicht auf der Startseite:
+jitsi_meet_default_background: "#C0C0C0"
+# By default it is an empty string because the CSS file is a bundled file for
+# the entire site, and it change very frequently with each release. It will be
+# replaced only if you have a custom CSS file and indicate its path in this variable
+jitsi_meet_css_file: 'jitsi.css'
+jitsi_meet_welcome_page_additions_file: jitsiMeetWelcomePage.html.j2
+jitsi_meet_default_remote_display_name: 'Mit dem Fahrrad da'
+
+jitsi_meet_configure_sip_gateway: false
```

Comment:
# Können wir das schon testen? Nummer haben wir ja noch aus dem SIP-Account, den wir für den SMS-Versand nutzen - oder köme sich das gegenseitig in die Quere? Ich kann auch Philpp aus Berlin fragen, der das anbieten wollte.

Reply:
Ja, prinizipiell schon, aber *bitte keine Passwörter* ins Git einschecken, die bekommt man dort nie wieder raus, schau mal im Pull-Request da habe ich ein Passwort  aus der Datei: /root/root.secret ausgelesen. So müssten wir das für das Sip auch machen

Reply:
Da wir noch keine Authentifizierung haben, kann dann jeder auf Kosten des ADFCs Gespräche aubauen. Also lieber nicht jetzt